src/apps/inventory/views.py

```diff
@@ -18,7 +18,7 @@ class ProductDetailView(DetailView):
     def get_queryset(self):
         category_slug = self.kwargs.get("category_slug")
         category = get_object_or_404(Category, slug=category_slug)
-        queryset = Product.objects.filter(category=category, is_active=True)
+        queryset = Product.objects.filter(category=category, is_active=True, stock__gt=0)
```

Comment:
We expect to return only product in stock? Will it be possible to see also out of stock product?

Reply:
Yep, lets keep only products in stock for now. u
Approved